File: lib_userinfo/src/main/java/com/caesarlib/userinfo/ValueUserData.java
package com.caesarlib.userinfo;

/**
 * created by Caesar on 2019/5/25 0025
 * email : 15757855271@163.com
 */
public class ValueUserData {
    private static String UserUuid = "";
    private static String UserToken = "";

    public static void setUserInfo(String userUuid, String userToken){
        UserUuid = userUuid;
        UserToken = userToken;
    }
    public static String getUserUuid() {
        return UserUuid;
    }


    public static String getUserToken() {
        return UserToken;
    }

}

File: lib_userinfo/src/main/java/com/caesarlib/userinfo/ValueUserData.kt
package com.caesarlib.userinfo

import androidx.databinding.ObservableField

/**
 * created by Caesar on 2019/5/25 0025
 * email : 15757855271@163.com
 */
object ValueUserData {
    val userUuid = ObservableField<String>("")
    val userToken = ObservableField<String>("")
    val nickName = ObservableField<String>("")
    val avatarImg = ObservableField<String>("")


    //当前的地址信息
    val province = ObservableField<String>()
    val city = ObservableField<String>()
    val area = ObservableField<String>()


    fun setUserInfo(userUuid: String?, userToken: String?) {
        this.userUuid.set(userUuid)
        this.userToken.set(userToken)
    }

}
